### PR TITLE
Setup 27

### PR DESCRIPTION
**Pull request by hannahli23**
- Added the semantic-ui-react dependency using yarn
- Added pages for home, map, email social worker, faq, more info
- Did a basic routing of those pages, formatting into hamburger style menu is another ticket
- Added an index.html age just for clarity

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,5 +16,10 @@
   "bugs": {
     "url": "https://github.com/team9353/food-deserts/issues"
   },
-  "homepage": "https://github.com/team9353/food-deserts#readme"
+  "homepage": "https://github.com/team9353/food-deserts#readme",
+  "dependencies": {
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "semantic-ui-react": "^0.88.2"
```
Comment (evan10s) [inline package.json]: I know this is a little confusing, but these dependencies should be added in the `client/package.json` file, not this one

**Review by evan10s — CHANGES_REQUESTED** [review]
looks good other than the little dependency issue.

don't forget to add "closes #27" to the body of the PR so it gets closed when you merge this

File: package.json (hunk 1 above)
Comment (hannahli23) [reply package.json]: Ahh okay thanks!

Comment (hannahli23) [reply package.json]: Looks like it's already in the client/package.json file too. I just deleted this from the main folder

**Review by hannahli23 — COMMENTED** [review]
Fixed the issue of dependency in package.json

**Review by evan10s — APPROVED** [review]
Looks good

**Comment by evan10s** [comment]
Don't forget "closes #27" in the PR description though

**Pull request merged**